### PR TITLE
UI updated and Azure MySQL connection done

### PR DESCRIPTION
**Pull request by wilsmic2**

### CODE REVIEW

File: server/tables.sql [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-CREATE TABLE IF NOT EXISTS PERMISSIONS (ROLE_ID INT PRIMARY KEY auto_increment, DESCRIPTION VARCHAR(255) NOT NULL UNIQUE);
-CREATE TABLE IF NOT EXISTS PEOPLE (PID INT PRIMARY KEY auto_increment, NAME VARCHAR(50) NOT NULL, ROLE_ID INT NOT NULL, VID INT NOT NULL, CONSTRAINT FK_Permission FOREIGN KEY (ROLE_ID) REFERENCES PERMISSIONS(ROLE_ID), CONSTRAINT FK_Village FOREIGN KEY (VID) REFERENCES VILLAGES(VID));
+CREATE TABLE IF NOT EXISTS PERMISSIONS (ROLE_ID INT PRIMARY KEY auto_increment, ROLE_TYPE VARCHAR(255) NOT NULL UNIQUE);
+CREATE TABLE IF NOT EXISTS PEOPLE (PID INT PRIMARY KEY auto_increment, FIRST_NAME VARCHAR(50) NOT NULL, LAST_NAME VARCHAR(50) NOT NULL, BIRTHDAY DATE, ROLE_ID INT NOT NULL, VID INT NULL, CONSTRAINT FK_Permission FOREIGN KEY (ROLE_ID) REFERENCES PERMISSIONS(ROLE_ID), CONSTRAINT FK_Village FOREIGN KEY (VID) REFERENCES VILLAGES(VID));
 CREATE TABLE IF NOT EXISTS ROOMS (ROOM_ID INT PRIMARY KEY auto_increment, ROOM_NUM INT NOT NULL, BLDG_NAME VARCHAR(50) NOT NULL, VID INT NOT NULL, VACANT BOOLEAN, CONSTRAINT FK_VID FOREIGN KEY (VID) REFERENCES VILLAGES(VID));
```
Comment (kushaljh) [inline server/tables.sql]: This needs to be updated as we just have house numbers and there are no buildings.

Its just village and house.

**Review by kushaljh — APPROVED** [review]
LGTM 

We should merge this branch in and then work on the comments in the future merges

File: server/tables.sql (hunk 1 above)
Comment (daniellee0) [reply server/tables.sql]: Fixed.

File: server/tables.sql [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+CREATE TABLE IF NOT EXISTS CREDENTIALS (CID INT PRIMARY KEY auto_increment, PID INT NOT NULL, EMAIL VARCHAR(255) NOT NULL, PASSWORD VARCHAR(255) NOT NULL)
+INSERT INTO VILLAGES (NAME) VALUES ("Interbay"), ("Whittier"), ("Othello"), ("Camp second chance"), ("Spirit"), ("Lake Union"), ("Plum st"), ("TEMS"), ("True Hope"), ("Georgetown"), ("Northlake");
+INSERT INTO PERMISSIONS (ROLE_TYPE) VALUES ('Employee'), ('Admin'), ('Resident');
+INSERT INTO PEOPLE (FIRST_NAME, LAST_NAME, BIRTHDAY, ROLE_ID, VID) VALUES ("Jennifer", "Smith", "1987-01-12", 3, 1), ("Sam", "Jackson", "1989-10-12", 3, 2);
+INSERT INTO PEOPLE (FIRST_NAME, LAST_NAME, ROLE_ID) VALUES ("Admin", "LIHI", 2);
+INSERT INTO CREDENTIALS (PID, EMAIL, PASSWORD) VALUES (3, "admin@lihi.org", "$2a$10$i20UIxsWH.w9P1lHaH34SudVOHFoaFISbMN0kwE1ClT5jxIRpIJxC");
+INSERT INTO HOUSES (HOUSE_NUM, VID, VACANT) VALUES (1, 1, TRUE), (1, 2, TRUE), (1, 3, TRUE), (1, 4, TRUE), (1, 5, TRUE), (1, 6, TRUE), (1, 7, TRUE), (1, 8, TRUE), (1, 9, TRUE), (1, 10, TRUE), (1, 11, TRUE);
```
Comment (daniellee0) [inline server/tables.sql:17]: All villages now have a default house "1". Use this for testing.

**Review by daniellee0 — APPROVED** [review]

**Pull request merged**